smoketests/tests/csharp_module.py

```diff
@@ -41,7 +41,7 @@ def test_build_csharp_module(self):
                 with open(csproj, "w") as f:
                     f.write(contents)
 
-                run_cmd("dotnet", "build", cwd=tmpdir, capture_stderr=True)
+                run_cmd("dotnet", "publish", cwd=tmpdir, capture_stderr=True)
```

Comment:
is this related to the PR contents, or separate optimization?

Reply:
It's related to testing this PR, see commit description. https://github.com/clockworklabs/SpacetimeDB/pull/1440/commits/79bb911c056324fcecc87a30c352cbc5b0264159

Reply:
oh neat! I had no idea.